packages/react-vis/package.json

```diff
@@ -104,7 +104,10 @@
         {
           "root": [
             "src"
-          ]
+          ],
+          "alias": {
+            "react-vis/*": "src/*"
+          }
```

Comment:
This seems incorrect? shouldn't this be in showcase?

Reply:
Theres a whole bunch of indirection happening with the references between these projects.  I'm trying to narrow it down.

Of course, these tests all work on my machine :P